packages/kit/types/ambient.d.ts

```diff
@@ -77,17 +77,17 @@ declare module '$app/environment' {
 	export const browser: boolean;
 
 	/**
-	 * Whether the dev server is running. This is not guaranteed to correspond to `NODE_ENV` or `MODE`.
+	 * SvelteKit analyses your app during the `build` step by running it. During this process, `building` is `true`.
```

Comment:
```suggestion
	 * SvelteKit analyses your app during the `build` step by running it. During this process, `building` is `true`.
	 * You can use this to wrap any code that should not be executed during build in a \`if (!building) {...}\` block.
```
Related: It might also be good to note in the page options docs that `building` will be `true` while prerendering. I also think it makes sense to explain somewhere how exactly the semantics of `building` are - for example, that this means top level code is run just because things are loaded, but that functions like `load` are not executed, unless at a later stage during prerendering.

Reply:
agree, added some stuff to the docs